airflow/cli/commands/variable_command.py

```diff
@@ -95,7 +96,7 @@ def _import_helper(filepath):
                 fail_count += 1
             else:
                 suc_count += 1
-        print("{} of {} variables successfully updated.".format(suc_count, len(var_json)))
+        print(f"{suc_count} of {len(var_json)} variables successfully updated.")
```

Comment:
From other changes I had the impression that "removing following period" is one of the the points you are enforcing in this PR. Maybe you missed this one?

Reply:
I tried, but then I resigned because I'm not sure what's better:
- `some text and 6` or `some text and 6.`
- `some 'text'=value` or `some 'text'=value.`
- 'some /text/and/path` or `some /text/and/path.`